src/libstd/fs.rs

```diff
@@ -844,10 +844,7 @@ impl OpenOptions {
         self
     }
 
-    /// Sets the option for creating a new file.
-    ///
-    /// This option indicates whether a new file will be created if the file
-    /// does not yet already exist.
+    /// Sets the option to create a new file, or open it if it already exists.
```

Comment:
I think this could be shortened to
```suggestion
    /// Sets the option to create a new file if it doesn't already exist.
```

Reply:
That's essentially what the comment previously said, but I thought it valuable to explicitly say what happens if the file exists, so a reader can easily contrast it with the description of what happens for `create_new` if the file exists.

Reply:
Yeah the change by the author looks good to me